arch/amd64-options.mk

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2018 Intel Corporation
```

Comment:
ohh, I seee, this is amd64 specific,
I wonder if an if sentence is needed in the makefile in order to include or not this file

Reply:
That's a fair point - branch updated with a comment ;)